src/providers/introduction.stories.mdx

```diff
@@ -128,17 +129,22 @@ const MyApp = () => {
     const data = await response.json();
     const meetingSessionConfiguration = new MeetingSessionConfiguration(data.Meeting, data.Attendee);
     const eventController = new DefaultEventController(
-        meetingSessionConfiguration, 
-        meetingManager.createLogger(meetingSessionConfiguration), 
-        new NoOpEventReporter()
-      );
+      meetingSessionConfiguration, 
+      meetingManager.createLogger(meetingSessionConfiguration), 
```

Comment:
I think `createLogger` should be a private method right @ltrung ?

Reply:
Done